src/en/charms-storage-ceph.md

```diff
@@ -0,0 +1,123 @@
+Title: Installing Ceph
+
+# Installing Ceph
+
+Here we present one way to install a rudimentary Ceph cluster with Juju backed
+by an AWS cloud. See [Ceph documentation][upstream-ceph-documentation] to find
+out more about Ceph.
```

Comment:
Have you reached out to @ChrisMacNaughton about any suggestions/best practice for the setup that might be worth including?